imports/startup/server/index.js

```diff
@@ -1,3 +1,4 @@
 import '/imports/api/messages.js';
 import '/imports/api/methods.js';
-import './startup.js';
+import './publications.js';
+import './routes.js';
```

Comment:
There is a different route.js in the server that should handle unknown addresses, throwing "404 not found" error. But it isn't working properly. It's a meteor bug, so I'm still working in a way to handle this.